cypress/e2e/check-characters-words-typos.cy.js

```diff
@@ -0,0 +1,18 @@
+describe("its going to test if the characters change, words, typos change as we write", () => {
```

Comment:
Maybe we can reword this to "how `karaktere, fjalë, shenjime` change as we write"?